cluster-provision/gocli/opts/cnao/manifests/operator.yaml

```diff
@@ -489,11 +517,12 @@ spec:
           capabilities:
             drop:
             - ALL
+        terminationMessagePolicy: FallbackToLogsOnError
       - args:
         - --logtostderr
         - --secure-listen-address=:8443
         - --upstream=http://127.0.0.1:8080
-        image: quay.io/openshift/origin-kube-rbac-proxy@sha256:e2def4213ec0657e72eb790ae8a115511d5b8f164a62d3568d2f1bff189917e8
+        image: quay.io/brancz/kube-rbac-proxy@sha256:e6a323504999b2a4d2a6bf94f8580a050378eba0900fd31335cf9df5787d9a9b
```

Comment:
Could you please elaborate why has this registry changed? (seems to be someones personal registry).
Same above and bellow.

Reply:
info here https://github.com/kubevirt/cluster-network-addons-operator/pull/1917
it is safe, this is the original repo https://github.com/openshift/kube-rbac-proxy is forked from